packages/@aws-cdk/cdk/lib/context.ts

```diff
@@ -34,6 +34,24 @@ export class Context {
     return new AvailabilityZoneProvider(scope).availabilityZones;
   }
 
+  /**
+   * Set the availability zone to be used in the given scope
```

Comment:
"zone" => "zones"

It's not "to be used", it's "that will be returned by `Context.getAvailabilityZones`